datasets/gov/vat/tikrinimo_vietos.csv

```diff
@@ -0,0 +1,19 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/vat/tikrinimo_vietos,,,,,,,,,,,,,"Pateikiama informacija apie prekių saugojimo ir muitinio tikrinimo vietas, pripažintas tinkamomis augalams, augaliniams produktams ir kitiems susijusiems objektams laikyti (pažymėjimo numeris; pažymėjimo išdavimo data; ūkio subjekto pavadinimas; sandėlio / terminalo tipas ir numeris; sandėlio / terminalo adresas) Pateikiama informacija apie Patikrinimo vietas, pripažintas tinkamomis atlikti fitosanitarinį tikrinimą įvežamiems į Lietuvos Respubliką ir Europos Sąjungą augalams, augaliniams produktams ir kitiems objektams laikyti (pažymėjimo numeris; pažymėjimo išdavimo data; ūkio subjekto pavadinimas; sandėlio / terminalo adresas)"
+,,,,,,prefix,spinta,,,,,https://github.com/atviriduomenys/spinta/issues/,,
+,,,,,,,manifest,,,,,https://github.com/atviriduomenys/manifest/issues/,,
+,,,,,,,,,,,,,,
+,,,,Muitine,,,eil_nr,,,4,open,,"Prekių saugojimo ir muitinio tikrinimo vietos, pripažintos tinkamomis augalams, augaliniams produktams ir kitiems susijusiems objektams laikyti ",
+,,,,,eil_nr,string,,,,4,open,,Eil. Nr.,
+,,,,,pazymejimo_isdavimas,date,D,,,4,open,,Pažymėjimo išdavimo data,
+,,,,,pazymejimo_nr,string,,,,4,open,,Pažymėjimo numeris,
+,,,,,terminalas,string,,,,4,open,,Sandėlio / terminalo adresas,
+,,,,,tipo_nr,string,,,,1,open,,Sandėlio / terminalo tipas ir numeris,
+,,,,,ukio_subjektas,ref,/datasets/gov/rc/jar/iregistruoti/JuridinisAsmuo[ja_pavadinimas],,,4,open,,Ūkio subjektas,
+,,,,,,,,,,,,,,
+,,,,FitosanitarineVieta,,,eil_nr,,,,,,"Patikrinimo vietos, pripažintos tinkamomis atlikti fitosanitarinį tikrinimą įvežamiems į Lietuvos Respubliką ir Europos Sąjungą augalams, augaliniams produktams ir kitiems objektams laikyti",
+,,,,,eil_nr,string,,,,4,open,,Eil. Nr.,
+,,,,,isdavimas,date,D,,,4,open,,Pažymėjimo išdavimo data,
+,,,,,pazymejimo_nr,string,,,,2,open,,Pažymėjimo numeris,
+,,,,,adresas,ref,/datasets/gov/rc/ar/adresotaskas/AdresoTaskas,,,4,open,,Sandėlio / terminalo adresas,
+,,,,,ukio_subjektas,ref,/datasets/gov/rc/jar/iregistruoti/JuridinisAsmuo[ja_pavadinimas],,,4,open,,Ūkio subjektas,
```

Comment:
```suggestion
,,,,,ukio_subjektas,ref,/datasets/gov/rc/jar/iregistruoti/JuridinisAsmuo[ja_pavadinimas],,,2,open,,Ūkio subjektas,
```
https://atviriduomenys.readthedocs.io/dsa/ref.html#susiejimas-nepatikimas

Be to, šių duomenų dar nėra IVPK Saugykloje.